utils.js

```diff
@@ -0,0 +1,21 @@
+export function getEndpoints() {
+  return fetch(`https://charliesdatabase.onrender.com/api`).then((res) => {
+    return res.json();
+  });
+}
+
+export function getReviews() {
+  return fetch(`https://charliesdatabase.onrender.com/api/reviews`).then(
+    (res) => {
+      return res.json();
+    }
+  );
+}
+
+export function getSingleReview(review_id) {
+  return fetch(
+    `https://charliesdatabase.onrender.com/api/reviews/${review_id}`
+  ).then((res) => {
+    return res.json();
+  });
+}
```

Comment:
Great to see you have a separate file for you api calls 🤸 I would however consider the name of your file. It isn't strictly speaking a util file but a file containing functions for each api call. I think this would be better in a utils folder with this as the api.js inside it 💭